src/app/ui/dashboard/NavLinks.tsx

```diff
@@ -1,20 +1,17 @@
-// Use when client needs JS
-"use client";
-
 import Link from "next/link";
 
-const links = [
+const linksData = [
```

Comment:
Its better to name this as `linksData` IMHO or something like that to not confused if its link as component, array of link or link item during the map rendering.

Also if you write it as Typescript (file has .tsx extension) you should add interface/type to this array to make it typesafe.. but it can be definitely done later :)

Reply:
linksData renamed.

TS used, but I guess there is some room for improvement:

1) icon should by defined as ReactComponent instead of any
2) is the use of inline types or interfaces preferred within such apps? 

P.S - You do not have to reply (but you can provide your personal experience / tips), purpose of these questions is to keep track of things i need learn / figure out.